base/commands/topic/topic_destroy.go

```diff
@@ -0,0 +1,66 @@
+//go:build base || topic
+
+package topic
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hazelcast/hazelcast-commandline-client/clc"
+	"github.com/hazelcast/hazelcast-commandline-client/errors"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/plug"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/prompt"
+	"github.com/hazelcast/hazelcast-go-client"
+
+	. "github.com/hazelcast/hazelcast-commandline-client/internal/check"
+)
+
+type topicDestroyCommand struct{}
+
+func (mc *topicDestroyCommand) Init(cc plug.InitContext) error {
+	long := `Destroy a Topic
+
+This command will delete the Topic and the data in it will not be available anymore.`
+	short := "Destroy a Topic"
+	cc.SetCommandHelp(long, short)
+	cc.AddBoolFlag(clc.FlagAutoYes, "", false, false, "skip confirming the destroy operation")
+	cc.SetCommandUsage("destroy [flags]")
```

Comment:
For simple commands, just setting the command name is equivalent to `cmd [flags]`
```suggestion
	cc.SetCommandUsage("destroy")
```